core/platform/android/java/src/dev/axmol/lib/AxmolRenderer.java

```diff
@@ -77,9 +77,12 @@ public void onSurfaceCreated(final GL10 GL10, final EGLConfig EGLConfig) {
         AxmolRenderer.nativeInit(this.mScreenWidth, this.mScreenHeight);
         this.mLastTickInNanoSeconds = System.nanoTime();
 
+        boolean isColdStart = mNativeInitCompleted;
```

Comment:
Is this correct? `mNativeInitComplete` is initially false, which is when the application is started for the first time, indicating a cold start. If it's true, it means that it has already been set in a previous run, which indicates that it is a warm start.  So, should this be:

`boolean isColdStart = !mNativeInitCompleted;`

Reply:
I was definitely not well waken up when I did this; you are right, I made a fix. After looking the better definition closer it's a Warm start so I renamed in consequence